docs/api-docs/storefront/widgets/widgets-tutorial.md

```diff
@@ -143,13 +137,13 @@ Accept: application/json
 
 ## Create a widget
 
-Now, use the widget template to create the widget. To do so, send a `POST` request to [`/v3/content/widgets`](https://developer.bigcommerce.com/api-reference/storefront/widgets-api/widget/createwidget).
+To create a widget, you will need to reference the widget template `uuid` from the previous step. Send a `POST` request to [`/v3/content/widgets`](https://developer.bigcommerce.com/api-reference/storefront/widgets-api/widget/createwidget) making sure to replace the placeholder value inside `widget_template_uuid` with your template's `uuid`.

```

Comment:
"To create a widget, use the widget template uuid from the previous step. Send a `POST` request to [`/v3/content/widgets`](https://developer.bigcommerce.com/api-reference/storefront/widgets-api/widget/createwidget) making sure to replace the placeholder value inside `widget_template_uuid` with your template's `uuid`.